plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -524,6 +523,16 @@ private String createContainer(String workspaceId,
                               .map(entry -> entry.getKey() + "=" + entry.getValue())
                               .toArray(String[]::new));
 
+        // If volume contains colon then it is bind volume, otherwise - pure volume.
+        hostConfig.setBinds(service.getVolumes()
+                                   .stream()
+                                   .filter(v -> v.contains(":"))
+                                   .toArray(String[]::new));
+        config.setVolumes(service.getVolumes()
```

Comment:
Don't you think that it makes more sense to go for each volume and make `contains(":")` check once?